### PR TITLE
Fix `IfNode#branches` to return both branches when called on ternary conditional

### PR DESCRIPTION
**Pull request by fatkodima**
Currently, it is returning only 1 branch.

### CODE REVIEW

**Comment by marcandre** [comment]
Good catch, thanks.

Expect a release later today.

File: spec/rubocop/ast/if_node_spec.rb [hunk 1]
```diff
@@ -434,6 +434,13 @@
       it { expect(if_node.branches).to all(be_literal) }
     end
 
+    context 'with a ternary operator' do
+      let(:source) { 'foo? ? :foo : 42' }
+
+      it { expect(if_node.branches.size).to eq(2) }
+      it { expect(if_node.branches).to all(be_literal) }
```
Comment (marcandre) [inline spec/rubocop/ast/if_node_spec.rb:441]: FYI, I think you can do both in a single expectaction `to match [be_literal, be_literal]`

Comment (fatkodima) [reply spec/rubocop/ast/if_node_spec.rb:441]: I just stick to the existing codestyle and replicated the nearest similar example 😄

**Comment by marcandre** [comment]
Oh, didn't notice, but the Changelog entry was misplaced.

Best use `rake changelog:...` in the future!

**Comment by marcandre** [comment]
Released 1.1.1, thanks!

**Comment by fatkodima** [comment]
I do not understand, how I did not noticed that it is not under master? It is the second time, at least, that I remember. 💦

**Comment by marcandre** [comment]
> I do not understand, how I did not noticed that it is not under master? It is the second time, at least, that I remember. 💦

You're probably too used to `rubocop` with a gazillion things in the "current" section

**Pull request merged**